modules/puzzle/src/main/Puzzle.scala

```diff
@@ -106,6 +104,7 @@ object Puzzle {
     val rating = s"$perf.gl.r"
     val vote = "vote"
     val voteSum = s"$vote.sum"
```

Comment:
since this field is no longer persisted in the database, it should also be removed from here. It will reveal compile errors that would otherwise be runtime errors.

Also, merge master back into that branch, I removed some hardcoded BSON fields to make puzzles more typesafe.